### PR TITLE
docs: add user documentation [DHIS2-7620]

### PR DESCRIPTION
**Pull request by tomzemp**
* add subsection for filter settings dialog (dashboard create mode)

* update relevant images to include Filter settings button

### CODE REVIEW

File: src/commonmark/en/content/user/managing-dashboards.md [hunk 1]
```diff
@@ -137,6 +137,20 @@ Click on **Exit Print preview** to return to editing the dashboard.
...
+
+By default, users will be able to filter dashboard items by any dimension defined in your system. Dashboard filter settings can be edited for a given dashboard by clicking on **Filter settings**.
+
+![](resources/images/dashboard/dashboard-filter-settings-button.png)
+
+To restrict available filters, you can click **Only allow filtering by selected dimensions** and select the filters which you wish to allow on the dashboard. When the dashboard is in view mode, users will only be able to choose from among the filters selected. Period and Organisation Unit will be selected by default but can be removed if desired.
+
+![](resources/images/dashboard/dashboard-filter-settings.png)
+
+In order to save updates to filter settings, you need to click **Confirm** to close the filter settings and then need to click **Save changes** to save the dashboard changes.
```
Comment (benguaraldi) [inline src/commonmark/en/content/user/managing-dashboards.md]: Let's change to:
> In order to save updates to filter settings, you need to first click **Confirm** to close the filter settings and then click **Save changes** to save the dashboard changes.

**Review by benguaraldi — COMMENTED** [review]
Looks good!  Just one minor change.

File: src/commonmark/en/content/user/managing-dashboards.md (hunk 1 above)
Comment (tomzemp) [reply src/commonmark/en/content/user/managing-dashboards.md]: Updated

Comment (benguaraldi) [reply src/commonmark/en/content/user/managing-dashboards.md]: Thanks!

**Review by benguaraldi — APPROVED** [review]

**Review by jimgrace — APPROVED** [review]

**Pull request merged**